policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/phase_out/percentage.yaml

```diff
@@ -0,0 +1,41 @@
+description: Rhode Island phases the standard deduction out at this rate, calculated based on modified federal AGI.
```

Comment:
Use the `0.2` instead from the legal code. This doesn't need a scale parameter. This will require `np.ceil` in the formula.